wagtail/admin/templates/wagtailadmin/pages/listing/_page_title_column_header.html

```diff
@@ -1,16 +1,23 @@
 {% extends "wagtailadmin/tables/column_header.html" %}
 {% load wagtailadmin_tags i18n %}
 
+<style>
+    /* Add a custom style block to adjust text baseline */
+    .baseline-adjust {
+        vertical-align: baseline;
+    }
+</style>
```

Comment:
We don't use inline `<style>` elements, please refer to our [UI guidelines](https://docs.wagtail.org/en/stable/contributing/ui_guidelines.html).

Putting the `<style>` element here also wouldn't work because this template extends a parent template. This is placed outside a `{% block %}`, so this element won't be rendered.

Even if it's moved to be inside a `{% block %}`, this still does not solve the issue:

<img width="1231" alt="image" src="https://github.com/wagtail/wagtail/assets/6379424/23030641-226d-43ec-93bb-0be7b41a95eb">


Please be sure to test your changes before submitting a PR. If you haven't set up a local development environment for Wagtail, follow our [Development](https://docs.wagtail.org/en/stable/contributing/developing.html) guide.

Reply:
We should be able to use the Tailwind class `w-align-baseline`
https://tailwindcss.com/docs/vertical-align

Reply:
Sorry, ignore that comment. Just re-read the comment here.